--- action_check.py     2017-10-09 13:35:49.851512365 +0000
+++ action_check.py.new 2017-10-09 13:38:30.657693872 +0000
@@ -425,7 +425,7 @@
           if not os.path.exists("/etc/systemd/system/multi-user.target.wants/rsyncd.service"):
              status.append(_("enable and start rsyncd.service with systemctl"))

-       if not os.path.exists("/usr/lib/systemd") and not os.path.exists("/etc/xinetd.d/rsync"):
+       if not os.path.exists("/usr/lib/systemd") and os.path.exists("/etc/xinetd.d/rsync"):
           status.append(_("file /etc/xinetd.d/rsync does not exist"))

    def check_dhcpd_conf(self,status):
[root@8d106da7113d cobbler]#
